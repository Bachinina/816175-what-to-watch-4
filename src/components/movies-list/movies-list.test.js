import React from "react";
import renderer from "react-test-renderer";
import MoviesList from "./movies-list.jsx";

const movies = [
  {
    src: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    title: `Fantastic Beasts: The Crimes of Grindelwald`
  },
  {
    src: `img/bohemian-rhapsody.jpg`,
    title: `Bohemian Rhapsody`
  },
  {
    src: `img/macbeth.jpg`,
    title: `Macbeth`
  },
  {
    src: `img/aviator.jpg`,
    title: `Aviator`
  }
];

it(`Should render MoviesList`, () => {
  const tree = renderer.create(
      <MoviesList
        movies={movies}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
