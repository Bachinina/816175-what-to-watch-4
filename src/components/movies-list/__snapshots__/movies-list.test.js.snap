// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render MoviesList 1`] = `
Array [
  <div
    className="catalog__movies-list"
  >
    <article
      className="small-movie-card catalog__movies-card"
      onMouseOver={[Function]}
    >
      <div
        className="small-movie-card__image"
        onClick={[Function]}
      >
        <img
          alt="Bohemian Rhapsody"
          height="175"
          src="img/bohemian-rhapsody.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
        onClick={[Function]}
      >
        <a
          className="small-movie-card__link"
        >
          Bohemian Rhapsody
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
      onMouseOver={[Function]}
    >
      <div
        className="small-movie-card__image"
        onClick={[Function]}
      >
        <img
          alt="Macbeth"
          height="175"
          src="img/macbeth.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
        onClick={[Function]}
      >
        <a
          className="small-movie-card__link"
        >
          Macbeth
        </a>
      </h3>
    </article>
  </div>,
  <div
    className="catalog__more"
  >
    <button
      className="catalog__button"
      type="button"
    >
      Show more
    </button>
  </div>,
]
`;
